src/main/java/com/opentok/MuteAllProperties.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * OpenTok Java SDK
+ * Copyright (C) 2021 Vonage.
+ * http://www.tokbox.com
+ *
+ * Licensed under The MIT License (MIT). See LICENSE file for more information.
+ */
+package com.opentok;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Defines values for the <code>properties</code> parameter of the
+ * {@link OpenTok#forceMuteAll(String, MuteAllProperties)} method.
+ *
+ * @see OpenTok#forceMuteAll(String, MuteAllProperties)
+ */
+public class MuteAllProperties {
+    private List<String> excludedStreamIds;
+
+    private MuteAllProperties(MuteAllProperties.Builder builder) {
```

Comment:
We also need methods related to the `active` option for this method. See the `active` option for the REST method: https://tokbox.com/developer/rest/#force_mute_session.

Reply:
added active option and tests.